src/framework/audio/internal/worker/itracksequence.h

```diff
@@ -0,0 +1,38 @@
+#ifndef MU_AUDIO_ITRACKSEQUENCE_H
```

Comment:
You need to add a license, and in other files too

Reply:
damn, I've completely forgotten it, thanks

fixed in - https://github.com/musescore/MuseScore/pull/8471/commits/1077547adede978049c6c39844395683f679e9af